esp/src/eclwatch/UserQueryWidget.js

```diff
@@ -867,10 +869,32 @@ define([
             registry.byId(this.id + "DeleteGroups").set("disabled", !hasGroupSelection);
             registry.byId(this.id + "ExportGroups").set("disabled", !hasGroupSelection);
 
+
             var permissionSelection = this.permissionsGrid.getSelected();
             var hasPermissionSelection = permissionSelection.length;
-            registry.byId(this.id + "EnableScopeScans").set("disabled", true);
-            registry.byId(this.id + "DisableScopeScans").set("disabled", true);
+            var scopeScansEnabled;
+            var scopeScansDisabled;
```

Comment:
Are two variables needed?  Also can they moved inside the "then" below since they are not referenced outside of that scope?